### PR TITLE
Update HTTP caching doc about zh-CN

### PR DESCRIPTION
**Pull request by eiis**
### Description

Update HTTP caching doc,Http headers etag,Http headers warning,Glossary cacheable about zh-CN

### Motivation

I think this way of translation is more in line with the habits and understanding of Chinese users

### CODE REVIEW

File: files/zh-cn/web/http/caching/index.md [hunk 1]
```diff
@@ -138,11 +138,11 @@ Age: 86400
 …
 ```
 
-收到该响应的客户端会发现它在剩余的 518400 秒内是新鲜（fresh）的，这是响应的 `max-age` 和 `Age` 之间的差异。
+收到该响应的客户端会发现它在剩余的 518400 秒内是有效的，这是响应的 `max-age` 和 `Age` 之间的差异。
 
 ## Expires 或 max-age
 
-在 HTTP/1.0 中，新鲜度过去由 `Expires` 标头指定。
+在 HTTP/1.0 中，有效期是通过 Expires 标头来指定的。
```
Comment (yin1999) [inline files/zh-cn/web/http/caching/index.md:145]:
```suggestion
在 HTTP/1.0 中，有效期是通过 `Expires` 标头来指定的。
```

File: files/zh-cn/web/http/caching/index.md [hunk 2]
```diff
@@ -219,7 +219,7 @@ Last-Modified: Tue, 22 Feb 2022 22:00:00 GMT
 Cache-Control: max-age=3600
 ```
 
-收到该响应后，客户端将存储的陈旧响应恢复为新鲜的，并可以在剩余的 1 小时内重复使用它。
+收到该响应后，客户端将存储的陈旧响应恢复为有效的，并可以在剩余的 1 小时内重复使用它。
```
Comment (yin1999) [inline files/zh-cn/web/http/caching/index.md:222]:
```suggestion
收到该响应后，客户端将存储的过期响应恢复为有效的，并可以在剩余的 1 小时内重复使用它。
```

**Review by yin1999 — COMMENTED** [review]
我赞成修改这里对 fresh 翻译的修改，这样的确更易于理解，但可以麻烦你更新前文中的相关翻译吗，当然，请保留第一处使用了该翻译的英文

```md
有效（fresh）
```

对应于过期，我觉得将 stale 翻译为“过期”会更加适合上下文语义。

@jasonren0403 你有啥想法吗

**Comment by fuguid** [comment]
Of course, I'd be happy to do that

File: files/zh-cn/web/http/headers/warning/index.md [hunk 3]
```diff
@@ -25,7 +25,7 @@ Warning: <warn-code> <warn-agent> <warn-text> [<warn-date>]
 
   - : 三位数字警告码。第一位数字表示 Warning 信息在验证之后是否需要从已存储的响应中删除。
 
-    - `1xx` 警告码描述了关于当前响应的新鲜度或者验证状态的警告信息，并且将会在验证之后被缓存服务器删除。
+    - `1xx` 警告码描述了关于当前响应的有效期或者验证状态的警告信息，并且将会在验证之后被缓存服务器删除。
```
Comment (yin1999) [inline files/zh-cn/web/http/headers/warning/index.md:28]: The English document does not match with the zh-CN document, I think there is a typo. Let's wait until mdn/content#28345 is resolved.

```suggestion
    - `1xx` 警告码描述了关于当前响应的有效期或者验证状态，并且将会在成功验证后被缓存服务器删除。
```

**Review by yin1999 — APPROVED** [review]
Thanks. I'll merge this after the update in upstream. Welcome aboard @fuguid 🎉

**Comment by yin1999** [comment]
The upstream changes have been merged. So let's go ahead :)

**Pull request merged**